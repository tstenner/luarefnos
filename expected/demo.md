<figure id="fig:1">
<img src="img/fig-1.png" style="width:1in" alt="Plot eins" />
<figcaption>Plot 1: Plot eins</figcaption>
</figure>

<figure id="fig:2">
<img src="img/fig-1.png" style="width:1in" alt="Plot zwei" />
<figcaption>Plot 2: Plot zwei</figcaption>
</figure>

Wir verweisen auf Plot [1](#fig:1) und Plots [1](#fig:1), die
[2](#fig:2) mit Postfix & fehlenden **fig:3?**.
