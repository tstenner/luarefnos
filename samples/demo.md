---
title: Pandoc-fignos Demo
lang: de
luarefnos:
  strs:
    de:
      tab: {ref: 'Tabelle', plural: 'Tabellen'}
      tbl: {ref: 'Tableu', plural: 'Tableus'}
      fig: {ref: 'Plot', abbrev: 'Plot'}
...

![Plot eins](img/fig-1.png){#fig:1 width=1in}

![Plot zwei](img/fig-1.png){#fig:2 width=1in}

Wir verweisen auf @fig:1 und [@fig:1; die @fig:2 mit Postfix; fehlenden @fig:3].

Foo Bar
--- ---
1   2

Table: Caption {#tbl:foobar}

Note that @tbl:foobar has a special namespace name.
