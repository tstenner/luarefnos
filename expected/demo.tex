\begin{figure}
\centering
\includegraphics[width=1in,height=\textheight]{img/fig-1.png}
\caption{Plot 1: Plot eins}\label{fig:1}
\end{figure}

\begin{figure}
\centering
\includegraphics[width=1in,height=\textheight]{img/fig-1.png}
\caption{Plot 2: Plot zwei}\label{fig:2}
\end{figure}

Wir verweisen auf Plot \ref{fig:1} und Plots \ref{fig:1}, die
\ref{fig:2} mit Postfix \& fehlenden \ref{fig:3}.

\begin{longtable}[]{@{}ll@{}}
\caption{Caption}\label{tbl:foobar}\tabularnewline
\toprule\noalign{}
Foo & Bar \\
\midrule\noalign{}
\endfirsthead
\toprule\noalign{}
Foo & Bar \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
1 & 2 \\
\end{longtable}

Note that Tableu \ref{tbl:foobar} has a special namespace name.
